library/core/src/primitive_docs.rs

```diff
@@ -1591,3 +1592,18 @@ impl<Ret, T> Clone for fn(T) -> Ret {
 impl<Ret, T> Copy for fn(T) -> Ret {
     // empty
 }
+
+// Fake impl that's only really used for docs.
+#[cfg(doc)]
+#[unstable(
+    feature = "fn_ptr_trait",
+    issue = "none",
+    reason = "internal trait for implementing various traits for all function pointers"
+)]
+#[doc(fake_variadic)]
+/// This trait is implemented on function pointers with any number of arguments.
```

Comment:
> function pointers with any number of arguments

 is probably better phrased as "all function pointers" for consistency and simplicity's sake